schema.sql

```diff
@@ -57,3 +57,13 @@ CREATE TABLE visits(
     FOREIGN KEY (vet_id) REFERENCES vets(id) ON DELETE RESTRICT ON UPDATE CASCADE,
     PRIMARY KEY (animal_id, vet_id, date)
 );
+
+/* Week 2 Day 1 */
+
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
```

Comment:
For query `SELECT * FROM visits where vet_id = 2`:
The first screenshot should use `Seq scan` and/or `Filter` instead of `Index scan`.
The execution time on the second screenshot should be significantly lower than 1 sec (1000 ms)